config/routes.rb

```diff
@@ -1,3 +1,8 @@
-Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-end
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  resources :posts
+  devise_for :users
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'posts#index'

```

Comment:
Making your application routes to use `resources :posts, only: [:new, :create, :index]` is part of the project requirements ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication)) Kindly update accordingly.